Exception/Exception.php

```diff
@@ -47,14 +47,3 @@
  */
 
 class Exception extends \Hoa\Core\Exception { }
-
-}
-
-namespace {
-
-/**
- * Flex entity.
- */
-Hoa\Core\Consistency::flexEntity('Hoa\File\Exception\Exception');
```

Comment:
Oh sorry, didn't get it :-). Excuse me. So yes, `flexEntity` prevents BC breaks by declaring a class as a “class entry”. For instance with `Hoa\Router` (because it happens recently):
```
Router/
    Router.php
    Http.php
    Cli.php
```
here, `Router.php` is a “class-entry” (its name is the same name as its parent directory). It means we can call `Hoa\Router` or `Hoa\Router\Router`, it's the same. Because the class inside `Router.php` has been defined as a flex entity.
But recently we moved `Http.php` to `Http/Http.php` to get:
```
Router/
    Router.php
    Http/
        Http.php
        Dav.php
    Cli.php
```
In this case, we declared the class in the `Http.php` file as a flex entity because this is a “class-entry” now (its name is the same than its parent directory name), and we can use `Hoa\Router\Http` like before, or `Hoa\Router\Http\Http`.

In all cases, this is **not** recommended to use the long/unfold name. This is unfolded by the autoloader (by catching unresolved class loading, adding the latest part if declared and re-running the stack of autoloaders).

It avoids BC breaks while keeping us free to change the structure a little bit in order to reorganize classes, interfaces and traits (so entities) as we want.